ts/components/session/LeftPaneContactSection.tsx

```diff
@@ -321,9 +321,15 @@ export class LeftPaneContactSection extends React.Component<Props, State> {
 
   private renderList() {
     const { sentFriendsRequest } = this.props;
+
+    const visibleFriendRequests = sentFriendsRequest.filter(
```

Comment:
Would be better to update the redux fetch directly (`sentFriendsRequest()` is not used anywhere else than in this file)
https://github.com/loki-project/session-desktop/blob/clearnet/ts/state/selectors/conversations.ts#L140

Also, if you change it here only, I think you have to update the `renderRow()` method too, no?

Reply:
Fixed